Source/WebKit/GPUProcess/graphics/WebGPU/RemoteXRSubImage.cpp

```diff
@@ -60,9 +60,24 @@ Ref<IPC::StreamServerConnection> RemoteXRSubImage::protectedStreamConnection()
     return m_streamConnection;
 }
 
+Ref<WebCore::WebGPU::XRSubImage> RemoteXRSubImage::protectedBacking()
+{
+    return m_backing;
+}
+
+Ref<RemoteGPU> RemoteXRSubImage::protectedGPU()
```

Comment:
Can't we mark this one `const` and omit the next `protectedGPU()` overload?
`WeakRef<T>::get()` is `const` and get returns a `T&`, like all our smart pointers.

Reply:
Will remove.